src/docker-osx-dev

```diff
@@ -441,7 +441,7 @@ function configure_docker_machine {
   DOCKER_HOST_NAME="$DOCKER_MACHINE_NAME"
   # Support < 0.4.1 and >= 0.5.1
   DOCKER_HOST_USER=$(inspect_docker_machine "{{.Driver.SSHUser}}" || inspect_docker_machine "{{.Driver.Driver.SSHUser}}")
-  DOCKER_HOST_IP=$(inspect_docker_machine "{{.Driver.IPAddress}}" || inspect_docker_machine "{{.Driver.Driver.IPAddress}}")
+  DOCKER_HOST_IP=$(docker-machine ip)
```

Comment:
It fallbacks to "default" if none is give: https://github.com/docker/machine/blob/5b9663f8960af73c048e14cc60566ae487c07728/commands/commands.go#L102 but to me it seems that it would be best to supply the `$DOCKER_MACHINE_NAME` in here so this would work for people using a different machine name :) So `$(docker-machine ip "$DOCKER_MACHINE")`